content/_index.en.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: OKFDE
-blurb:  Open Knowledge for a Digital Society
-work: We fight for open knowledge and use technology in the interests of civil society. Find out exactly what we do and what drives us.
-donate_title: Support us now!
+blurb:  Open knowledge for a digital civil society
+work: We fight for open knowledge and democratic participation by developing technologies and tools that strengthen civil society.
```

Comment:
**Other suggestions:**
+ A strong democracy and an open society, using digital technologies.
+ We want to see a digital civil society for more participation and empowerment.
+ Bringing government transparency and citizen participation to the digital world.

Reply:
Since this strongly relates to the main claim I would argue to stick with the most literal translation (I agree with your in-line suggestion). The ones you commented are good inspirations for future discussions :)